### PR TITLE
June 2020 meeting minutes

### PR DESCRIPTION
**Pull request by Harteex**
Meeting minutes for June is finally here :)

### CODE REVIEW

File: content/project-news/june-2020-meeting-minutes.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+
+He reported that it's almost ready.
+Foxlet provided all the press media kit screenshots with his design.
+What's left is to bring the press document to Zachary for English grammar corrections and such.
+
+## Point 5: Unifying our contribution guidelines and handling Pull Requests
+For the last agenda point, Victor had already created a document outlining his thoughts.
+He gave a quick recap in the meeting channel about the two basic goals he wanted to achieve:
+
+* Reduce the decision time: reject PR or not, and reduce overall review time of course
```
Comment (learn-more) [inline content/project-news/june-2020-meeting-minutes.md:64]: `of course` feels out of place here,
do you intend it to 'review the time of the course' or 'of course we want to reduce the overall review time' ?
For the former, 'of the course' might be better, for the latter, I'd drop 'of course'

Comment (Harteex) [reply content/project-news/june-2020-meeting-minutes.md]: Yes, I agree. Those lines were taken as-is from Victor's recap. It sounds like he meant the latter, so I dropped it. 👍

Comment (Extravert-ir) [reply content/project-news/june-2020-meeting-minutes.md]: Yeah, I guess I missed an article here

File: content/project-news/june-2020-meeting-minutes.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+
+Colin agreed that we simply lack manpower to handle every trivial Pull Request, and Mark wrote that having clear contribution guidelines would help in shooting down those PRs.
+
+We already have CONTRIBUTING.md and PULL_REQUEST_MANAGEMENT.md (the latter which was decided during a previous meeting).
+On the question which guidelines in the tree the new document should replace, Victor responded that it probably should be merged with CONTRIBUTING.md.
+
+Some of the suggestions in the document was discussed.
+One of the points suggested in the document was a limit on the number of open PRs, but most disagreed with this change.
+
+Discussed next was code style.
```
Comment (Extravert-ir) [inline content/project-news/june-2020-meeting-minutes.md:75]: I feel there is something wrong here from the grammar point of view. Should the noun be in the beginning?

Comment (Harteex) [reply content/project-news/june-2020-meeting-minutes.md:75]: I think it is correct, but I'm not a native speaker. The same kind of sentence seems to be used at other places too.

Comment (learn-more) [reply content/project-news/june-2020-meeting-minutes.md:75]: This seems correct in an informal context.

Comment (Harteex) [reply content/project-news/june-2020-meeting-minutes.md:75]: Alright. Also, other text (including other meeting summaries I've found) uses the same construct. Resolving.

**Pull request merged**